Main.java

```diff
@@ -9,7 +10,21 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+// create a Scanner for user input
+  Scanner input = new Scanner(System.in);
+
+  // declare a variable for the first number
+
+  // ask the user for a number
+   System.out.println("Please enter a number:");
+   // initialize a number that needs to be divided by another number
+
+   // declare and initialize a variable for integer1
+   int number1 = input.nextInt();
+
+   // declare and initialize a variable for integer1
+   int number2 = input.nextInt();
+   // declare and initialize the quotient from the 2 numbers
+  System.out.println("The answer is " + (number1%number2));
   }

```

Comment:
This is a good start but you are also missing the quotient (the answer when you divide). Also, we try to avoid putting math statements inside of the print command. Instead, create a variable that you can use to store that math and then use that variable inside of the print statement.

int remainder = number1 % number2;